### PR TITLE
Implement List Keystores for Standard API

### PR DESCRIPTION
**Pull request by rauljordan**
This PR implements the list keystores method for the validator standard key management API as defined in the schema [here](https://github.com/prysmaticlabs/prysm/blob/ad9e5331f53b781977fc3994b814bc9253991a7a/proto/eth/service/key_management.proto). It does not wrap it with an API middleware yet and is simply a grpc implementation of the endpoint with tests

### CODE REVIEW

File: validator/rpc/standard_api.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	pubKeys, err := s.keymanager.FetchValidatingPublicKeys(ctx)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Could not list keystores: %v", err)
+	}
+	keystoreResponse := make([]*ethpbservice.ListKeystoresResponse_Keystore, len(pubKeys))
+	for i := 0; i < len(pubKeys); i++ {
+		keystoreResponse[i] = &ethpbservice.ListKeystoresResponse_Keystore{
+			ValidatingPubkey: pubKeys[i][:],
+		}
+		if s.wallet.KeymanagerKind() == keymanager.Derived {
```
Comment (rauljordan) [inline validator/rpc/standard_api.go:31]: Note, this is a violation of the open/closed principle as discussed in https://www.notion.so/prysmaticlabs/Open-Closed-Principle-Applied-to-Keymanagers-900393aa4cb243f5aead5578c8e801d7. However, fixing this would require a refactor of the keymanager interface which shall be done in a separate PR

**Review by rkapka — APPROVED** [review]
